Engine/platform/osx/acplmac.cpp

```diff
@@ -113,6 +151,31 @@ void AGSMac::ShutdownCDPlayer() {
   //cd_exit();
 }
 
+const char* AGSMac::GetAllUsersDataDirectory()
+{
+  return commonDataPath;
+}
+
+const char *AGSMac::GetUserSavedgamesDirectory()
+{
+  return libraryApplicationSupport;
+}
+
+const char *AGSMac::GetUserConfigDirectory()
+{
+  return libraryApplicationSupport;
+}
+
+const char *AGSMac::GetAppOutputDirectory()
+{
+  return commonDataPath;
+}
+
+const char *AGSMac::GetIllegalFileChars()
+{
+  return "\\/:?\"<>|*"; // keep same as Windows so we can sync.
```

Comment:
To elaborate my confusion: why does it matter for, for example, OSX port which paths are used by Windows version?